src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/UsersRequestExecutor.java

```diff
@@ -129,10 +130,10 @@ public static Response countGroupByTag(HttpServletRequest servletRequest,
     SortedMap<OSHDBCombinedIndex<OSHDBTimestamp, Pair<Integer, Integer>>, Integer> result = null;
     result = mapRed.flatMap(f -> {
       List<Pair<Pair<Integer, Integer>, OSMContribution>> res = new LinkedList<>();
-      int[] tags = ExecutionUtils.extractContributionTags(f);
-      for (int i = 0; i < tags.length; i += 2) {
-        int tagKeyId = tags[i];
-        int tagValueId = tags[i + 1];
+      Iterable<OSHDBTag> tags = ExecutionUtils.extractContributionTags(f);
+      for (OSHDBTag tag : tags) {
+        int tagKeyId = tag.getKey();
+        int tagValueId = tag.getValue();
         if (tagKeyId == keysInt) {
           if (valuesInt.length == 0) {
             res.add(new ImmutablePair<>(new ImmutablePair<>(tagKeyId, tagValueId), f));
```

Comment:
Same here, we could use `OSHDBTag tag` here instead of creating a new Pair of `new ImmutablePair<>(tagKeyId, tagValueId)`.